app/components/repo-actions.js

```diff
@@ -1,7 +1,7 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
 import { computed } from '@ember/object';
-import { alias, and, or } from '@ember/object/computed';
+import { alias, and, or, reads } from '@ember/object/computed';
```

Comment:
This change seems to be unnecessary
```suggestion
import { alias, and, or} from '@ember/object/computed';
```

Reply:
Removed.